torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -20,15 +20,15 @@ def horizontal_flip_bounding_box(bounding_box: torch.Tensor, *, image_size: Tupl
         old_format=BoundingBoxFormat.XYXY,
         new_format=BoundingBoxFormat.XYWH,
     ).unbind(-1)
-    x = image_size[1] - (x + w)
+    x = image_size[1] - (x + w)  # Why not avoid formatting and do straight `boxes[:, [0, 2]] = width - boxes[:, [2, 0]]`?
```

Comment:
That would modify inplace so we need to do `boxes = boxes.clone()` first. Otherwise I have no preference over one or the other.

Reply:
Yeah, I'm not advocating inplace modifications of the input. Just a simplification of the transform.